examples/preact/size-plugin.json

```diff
@@ -0,0 +1 @@
+[{"timestamp":1642471583231,"files":[{"filename":"ssr-build/ssr-bundle.43203.css","previous":569,"size":0,"diff":-569},{"filename":"ssr-build/ssr-bundle.js","previous":4555,"size":0,"diff":-4555},{"filename":"bundle.4922e.css","previous":0,"size":447,"diff":447},{"filename":"bundle.*****.esm.js","previous":0,"size":8322,"diff":8322},{"filename":"polyfills.*****.esm.js","previous":0,"size":2191,"diff":2191},{"filename":"route-home.chunk.8aeee.css","previous":0,"size":75,"diff":75},{"filename":"route-home.chunk.*****.esm.js","previous":0,"size":252,"diff":252},{"filename":"route-profile.chunk.141aa.css","previous":0,"size":78,"diff":78},{"filename":"route-profile.chunk.*****.esm.js","previous":0,"size":1137,"diff":1137},{"filename":"sw-esm.js","previous":0,"size":10767,"diff":10767},{"filename":"sw.js","previous":0,"size":10762,"diff":10762},{"filename":"bundle.8eca2.js","previous":0,"size":8369,"diff":8369},{"filename":"polyfills.03377.js","previous":0,"size":2291,"diff":2291},{"filename":"route-home.chunk.732e3.js","previous":0,"size":256,"diff":256},{"filename":"route-profile.chunk.6a5ba.js","previous":0,"size":1625,"diff":1625},{"filename":"index.html","previous":0,"size":1045,"diff":1045},{"filename":"200.html","previous":0,"size":645,"diff":645}]}]
```

Comment:
TBH I don't really understand what this is, but it was included in the hello world template from Preact.